### DISCUSSION TITLE
Decouple built-in @directus/app components into individual packages

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#14479] [Feature Requests]
Whenever I try to create a modified version of a built-in panel/interface/layout/module/etc., I would usually scaffold a new package and copy-paste the source code from @directus/app. Unfortunately, this is a difficult task since the built-in components have internal dependencies within @directus/app. Would love to see these being decoupled since these will also serve as real-world guides in trying to customize the directus app

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
